react/features/welcome/components/WelcomePage.web.js

```diff
@@ -285,7 +285,11 @@ class WelcomePage extends AbstractWelcomePage {
             return null;
         }
 
-        const { _calendarEnabled, t } = this.props;
+        const { _calendarEnabled, _recentListEnabled, t } = this.props;
+
+        if (!_calendarEnabled && !_recentListEnabled) {
+            return null;
```

Comment:
Instead, please check if tabs.lengs is 0 and return null later on. This will make it more future proof, in case we add more tabs.